### ISSUE TITLE
heroku:add_deploy_hook task doesn't handle multiple Heroku apps

### ISSUE THREAD

**Issue by Dorian** [bugsnag/bugsnag-ruby#196]
eg, I have four apps and I get:

```
> bin/rake bugsnag:heroku:add_deploy_hook
 !    Multiple apps in folder and no app specified.
 !    Specify app with --app APP.
$ heroku addons:add deployhooks:http --url="https://notify.bugsnag.com/deploy?apiKey=SECRET_STUFF&branch=master&revision={{head_long}}&releaseStage=development&repository=https://github.com/SOMETHING"
 !    Multiple apps in folder and no app specified.
 !    Specify app with --app APP.
```

**Comment by Dorian** [comment]
Maybe it's supposed to be run as `heroku run bin/rake bugsnag:heroku:add_deploy_hook -a MYAPP`?

**Comment by loopj** [comment]
Hey @Dorian, this is fixed in #198 and will be released soon

**Comment by snmaynard** [comment]
I believe this is fixed! Let me know if it isnt!
